llama-index-integrations/vector_stores/llama-index-vector-stores-milvus/llama_index/vector_stores/milvus/base.py

```diff
@@ -45,11 +47,28 @@
 def _to_milvus_filter(standard_filters: MetadataFilters) -> str:
     """Translate standard metadata filters to Milvus specific spec."""
     filters = []
-    for filter in standard_filters.legacy_filters():
-        if isinstance(filter.value, str):
-            filters.append(str(filter.key) + " == " + '"' + str(filter.value) + '"')
+    for filter in standard_filters.filters:
+        filter_value = (
+            f'"{filter.value!s}"'
+            if isinstance(filter.value, str)
+            else str(filter.value)
+        )
+        if isinstance(filter.operator, FilterOperatorFunction):
+            operator = filter.operator.value.format(key=filter.key, value=filter_value)
+            filters.append(operator)
         else:
-            filters.append(str(filter.key) + " == " + str(filter.value))
+            match filter.operator:
```

Comment:
hmm, pretty sure the match syntax was only introduced in 3.10 or 3.9

We should support 3.8+ where possible -- lets make this an if/elif chain

Reply:
This is done